src/applications/yellow-ribbon/containers/YellowRibbonApp/index.jsx

```diff
@@ -0,0 +1,108 @@
+// Node modules.
+import React from 'react';
+import Breadcrumbs from '@department-of-veterans-affairs/formation-react/Breadcrumbs';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+// Relative imports.
+import SearchForm from '../SearchForm';
+import SearchResults from '../SearchResults';
+
+export const YellowRibbonApp = ({ results }) => (
+  <>
+    {/* Breadcrumbs */}
+    <Breadcrumbs customClasses="vads-u-padding--0">
+      <a href={`${window.location.origin}`} rel="noopener noreferrer">
+        Home
+      </a>
+      <a
+        href={`${window.location.origin}/education/`}
+        rel="noopener noreferrer"
+      >
+        Education and training
+      </a>
+      <a
+        href={`${window.location.origin}/yellow-ribbon/schools/`}
+        rel="noopener noreferrer"
+      >
+        Yellow Ribbon school finder
+      </a>
+      {results && (
+        <a href={window.location.href} rel="noopener noreferrer">
```

Comment:
Agreed... technically the last one isn't even a link since Breadcrumbs always modifies it. I think this will be moot with the comparison PR though as it will need to be updated.